lib/column/ipv4.go

```diff
@@ -51,6 +51,11 @@ func (col *IPv4) Row(i int, ptr bool) interface{} {
 
 func (col *IPv4) ScanRow(dest interface{}, row int) error {
 	switch d := dest.(type) {
+	case *string:
+		*d = col.row(row).String()
+	case **string:
```

Comment:
where is the need for support for this coming from? can you give me an example as to when we need  **string support?

Reply:
I relied on String column's ScanRow method.

https://github.com/ClickHouse/clickhouse-go/blob/a27ceead7ef92930e47841cec6d08cc24912c4dd/lib/column/string.go#L61

Reply:
yes i noticed this, let me see if that's actually something useful..i might be missing a use case.